src/common/controllers/ads-controller.js

```diff
@@ -178,6 +201,7 @@ class AdsController extends FakeEventTarget implements IAdsController {
       });
     if (this._configAdBreaks.length) {
       this._dispatchAdManifestLoaded();
+      this._handlePrebidAdConfig();
```

Comment:
why we need to do it in advance and not when playAdNow actually called? this way you support also playAdNow as api

Reply:
do you want to use playAdNow with prebid?? I didn't think it's in scope.
I'm loading it in advance because ad loading could take around x seconds of a timeout then for seeks, it will work much more smoothly. also if we'll do it for each playAdNow it says it'll play an ad that configured in position 10 between 10 to 10 + timeout, instead of playing each ad at the time.

Reply:
add implementation for playAdNow

Reply:
done